release-schema: Add finalStatus to tender, award and contract. Clarify tender.datePublished. (#1648)
open-contracting/standard: docs/guidance/build/change_history.md

The publisher creates a new OCDS release for the award. The record has the relevant information in the award section and uses the 'award' tag. 

> The `parties` array has a new entry with the supplier's information. The complete `tender` section is repeated. Previous releases remain unchanged.

Reviewer: Otherwise, the reference to `tender.finalStatus` in the next paragraph doesn't work.

```suggestion
The `parties` array has a new entry with the supplier's information. The complete `tender` section is repeated, and its `finalStatus` field is set to 'complete'. Previous releases remain unchanged.
```

> The publisher adds the new release to the record. They also update the compiled and versioned releases. The compiled release reflects the changes to the `tender`, `awards` and `parties` sections. The versioned release includes a new change for the `tender.finalStatus` field.

Reviewer: ```suggestion
The publisher adds the new release to the record. They also update the compiled and versioned releases. The compiled release reflects the changes to the `tender`, `awards` and `parties` sections. The versioned release now sets the `tender.finalStatus` field.
```
